intact-orthology-import/src/main/java/uk/ac/ebi/intact/ortholog/OrthologsProteinAssociation.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.ebi.intact.ortholog;
+
+import uk.ac.ebi.intact.jami.dao.IntactDao;
+import uk.ac.ebi.intact.jami.model.extension.IntactProtein;
+
+import javax.annotation.Resource;
+import java.util.*;
+import java.util.logging.Logger;
+
+public class OrthologsProteinAssociation {
+
+    private static final Logger logger = Logger.getLogger(OrthologsProteinAssociation.class.getName());
+
+    @Resource(name="intactDao")
+    private final IntactDao intactDao;
+
+    public OrthologsProteinAssociation(IntactDao intactDao) {
+        this.intactDao = intactDao;
+    }
+
+    public Collection<IntactProtein> getIntactProtein(){
+        logger.info("Fetching all Intact Proteins...");
+        return intactDao.getProteinDao().getAll();
+    }
+
+    public Collection<IntactProtein> flattenProteinCollection(Collection<Collection<IntactProtein>> nestedProteins) {
+        List<IntactProtein> flatProteins = new ArrayList<>();
+        for (Collection<IntactProtein> proteinCollection : nestedProteins) {
+            flatProteins.addAll(proteinCollection);
+        }
+        return flatProteins;
+    }
+
+    public Collection<IntactProtein> getFewIntactProtein() {
```

Comment:
Is this just for testing?

Reply:
Yes it is to try on 10 proteins instead of the whole db